readme.md

```diff
@@ -65,12 +70,14 @@ Describe the purpose of views and models in FE JS.
 
 Your Answer:
 ```text
-
+Well, views are a way of getting information to show up on the client side (browser); models are a way of structuring the data sets and giving them a form so
+they all evaluate in a similar way.
 ```
```

Comment:
Close. Models are responsible for fetching and storing data. Views are then responsible for not only rendering that data but defining any app functionality that relates to user interactions (e.g., event listeners).